modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -15,6 +15,7 @@
 #include <string>
 #include <limits>
 #include <algorithm>
+#include <strstream>
```

Comment:
`<sstream>` ?

Reply:
Yup.  I've dropped that line and verified that it still builds.